news/2023-01-01-osumonthly-6.md

```diff
@@ -0,0 +1,331 @@
+
+## Things to look out for in January
+
+- Yet another LAN tourney is on the horizon. Organized by [Zeus-](https://osu.ppy.sh/users/5464437) with the support of [Boğaziçi University Gaming Club (BUOK)](http://buok.boun.edu.tr/), the very first iteration of **Bosphorus Invitational** aims to gather around some of the best circle clickers within the Turkish community at the border of the magnificent Bosphorus strait where the continents meet, for an epic 1 day showdown of Turkey's best players. While the participation is invite-only, feel free to either visit the venue on the 29th or check out the tournament via the [BUOK Twitch channel](https://www.twitch.tv/buoyunkulubu/) to keep the tabs on. See more on the details through the [forum post](https://osu.ppy.sh/community/forums/topics/1696844), which is also [available in Turkish](https://osu.ppy.sh/community/forums/topics/1696848)!
+- The [New Beginnings Art Contest](https://osu.ppy.sh/home/news/2022-12-24-new-beginnings-voting-open) voting has concluded by the time you are reading this, and should have it's results posted. Go and check out all of the wonderful creations from the community, and if you previously put your votes in, check if the creations you voted for have made their way to the top!
+- For those of you who are ***Full Beatmap Nominators*** and have an interest in being a part of the [NAT](/wiki/People/The_Team/Nomination_Assessment_Team), they are going to be doing some specific osu! BN evaluating where the full BNs are able to "play NAT for a bit" from January 1 - 31, 2023. If you are interested, you might want to talk to your local NAT representative.
+- Qualifiers for the [osu!mania 7K World Cup](https://osu.ppy.sh/home/news/2022-12-15-osu-mania-7k-world-cup-2023-registrations-now-open) are starting on the 8th of January, so be sure to tune in if you are a fan of top tier mania gameplay!
+- During January and February the [4 Digit 4K MWC 2023 tournament](https://osu.ppy.sh/community/forums/topics/1671710?n=1) will be held, and while the gameplay might not be as top tier as the official World Cup, mania enjoyers might find themselves highly interested in this tournament as well.
+
+## End word
+
+As always, thank you for reading this month's edition of osu!monthly. We hope that 2022 treated you well and that the new year brings even more splendors. If you have any questions, comments, concerns, or want to contribute don't hesitate to say something in the osu!dev discord. We thank you for an amazing reponse for half of the year and we hope this continues on for many more, it is not without you all creating community content that this cannot happen. Thank you, thank you, thank you!
```

Comment:
```suggestion
As always, thank you for reading this month's edition of osu!monthly. We hope that 2022 treated you well and that the new year brings even more splendors. If you have any questions, comments, concerns, or want to contribute don't hesitate to say something in the [osu!dev Discord server](https://discord.com/invite/ppy). We thank you for the amazing response to osu!monthly during the last half of a year and we hope this continues on for many more years, it is with you all creating community content that this can even happen. Thank you, thank you, thank you!
```